.github/workflows/test_pr.yml

```diff
@@ -53,6 +53,7 @@ jobs:
       - name: Test
         run: |
           python -m pytest -k test_basic --color=yes --cov-report=xml --cov=acquire --maxfail=5 --log-cli-level=0
+          python -m pytest -k test_docs --color=yes --cov-report=xml --cov=acquire --maxfail=5 --log-cli-level=0
```

Comment:
Is there a difference between explicitly collecting doctests with `test_docs` and using the pytest doctest options?

If not, I think we should make these consistent (ideally through the pytest options in the pyproject.toml file). See other comments for more context.